src/components/GoodsList/GoodsList.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './GoodsList.scss';
+
+type Props = {
+  goods: Good[]
+};
+
+export const GoodsList: React.FC<Props> = ({ goods }) => (
+  <ul className="goods__list">
+    {goods.map((good) => (
+      <li
+        className="goods__item"
+        style={{ color: `${good.color}` }}
```

Comment:
```suggestion
        style={{ color: good.color }}
```
it turns out that it is possible)